js/components/frame.js

```diff
@@ -121,9 +121,11 @@ class Frame extends ImmutableComponent {
 
     let partition
     if (this.props.frame.get('isPrivate')) {
-      partition = 'private-1'
+      partition = 'default'
     } else if (this.props.frame.get('partitionNumber')) {
       partition = `persist:partition-${this.props.frame.get('partitionNumber')}`
+    } else {
+      partition = 'persist:default'
```

Comment:
Does this cause a different partition name for the default partition? In particular people that have a gmail account setup for example will still have that setup right?